static/css/page-book.less

```diff
@@ -28,6 +28,7 @@
 
 @import (less) "components/metadata-form.less";
 @import (less) "components/observationStats.less";
+@import "components/buttonBtn.less";
```

Comment:
```suggestion
@import (less) "components/buttonBtn.less";
```

Reply:
According to the update on https://github.com/internetarchive/openlibrary/issues/4933#issuecomment-809646182 the component should be imported by `header.less` but importing `components/buttonBtn.ess` directly solved the issue 🎉

Reply:
All okay now?